src/code/stores/graph-store.coffee

```diff
@@ -524,7 +523,7 @@ GraphStore  = Reflux.createStore
           transfer = link.transferNode
           modelDescription += "#{transfer.key}:#{transfer.initialValue}:#{transfer.combineMethod};" if transfer
         modelDescription += "#{target.key}#{if target.isAccumulator then ':'+(target.value ? target.initialValue) else ''}|"
-
+        modelDescription += target.combineMethod
```

Comment:
It seems to be customary to put a delimiter at the end of each chunk. Otherwise, the `combineMethod` would be concatenated with the subsequent node count.

Reply:
Good catch!  -- Addressed in 086e106